app/views/content/non-uk-teachers/visas-for-non-UK-teachers.md

```diff
@@ -0,0 +1,179 @@
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+---
+
+If you’re applying for teaching jobs in England as a non-UK citizen, you will need a visa or immigration status allowing you to work in England. 
+
+The information on this page is for:
+
+* non-UK citizens living in the UK
+* non-UK citizens living outside the UK
+* refugees or asylum seekers
+
+### Teachers who will not need a visa
+
+You will not need a visa if you:
+
+* are an Irish citizen
+* have [settled or pre-settled status under the EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
+* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk)
+
+If you are currently studying in the UK on a student visa, you may be able to apply for a [graduate visa](#other-types-of-visa-you-can-use-to-work-as-a-teacher-in-england).
+
+If you or your family are from the European Economic Area (EEA) or Switzerland, and you or a family member were living in the UK by 31 December 2020, you may still be able to [apply to the EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families).
+
+## Check your status
+
+If you’re not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa
+).
+
+You can also [contact UK Visas and Immigration for help](https://www.gov.uk/contact-ukvi-inside-outside-uk).
+
+## The skilled worker visa for teachers
+
+The main visa route for non-UK citizens wishing to teach in England is the skilled worker visa.
+
+You’ll be able to apply for your skilled worker visa if:
+
+* you have a job offer from a school that is a Home Office licensed visa sponsor 
+* [you can speak, read, write, and understand English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
+* your salary meets the requirements for a skilled worker visa
+
+[Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas)
+
+### Salary requirement for a skilled worker visa
+
+The salary you must be paid to get a visa depends on whether you have [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts#teachers-who-trained-outside-the-uk) and the region you are applying to work in.
+
+The minimum salary non-UK teachers must be paid to be eligible for a skilled worker visa are as follows.
```

Comment:
should this either be 'the minimum salary ... **is** as follows' or 'the minimum **salaries** ... are as follows'

Reply:
Yes!